### PR TITLE
Sw1.5/issue 24 prep work

### PR DESCRIPTION
**Pull request by henryaddison**
Some tweaks to how type references are validated

### CODE REVIEW

File: src/models-spec.js [hunk 1]
```diff
@@ -103,15 +114,23 @@ describe('models', () => {
...
+              result.pass = skosDataModel.includes(typeId);
+              if (!result.pass) {
+                result.message = `${typeRef} is not a valid goodrelations reference`;
+              }
             } else if (typeId.match(/^https:\/\/openactive.io/)) {
               const typeName = typeId.replace(/^https:\/\/openactive.io\//, '');
               const enums = getEnums(version);
               result.pass = modelExists(typeName) || Object.prototype.hasOwnProperty.call(enums, typeName);
               if (!result.pass) {
-                result.message = `${typeRef} is not a valid Open Active reference`;
```
Comment (nickevansuk) [inline src/models-spec.js]: Note OpenActive never has a space between “Open” and “Active”

File: src/models-spec.js [hunk 2]
```diff
@@ -103,15 +114,23 @@ describe('models', () => {
...
               }
+            } else if (typeId.match(/^http:\/\/purl\.org\/goodrelations\/v1/)) {
+              result.pass = goodRelationsDataModel.includes(typeId);
+              if (!result.pass) {
+                result.message = `${typeRef} is not a valid goodrelations reference`;
+              }
+            } else if (typeId.match(/^http:\/\/www\.w3\.org\/2004\/02\/skos\/core#/)) {
+              result.pass = skosDataModel.includes(typeId);
+              if (!result.pass) {
+                result.message = `${typeRef} is not a valid goodrelations reference`;
```
Comment (nickevansuk) [inline src/models-spec.js]: Is this error message wrong? Should be SKOS?

File: src/models-spec.js (hunk 1 above)
Comment (henryaddison) [reply src/models-spec.js]: Bad Henry. Thanks, @nickevansuk. Fixing this in later PR.

File: src/models-spec.js (hunk 2 above)
Comment (henryaddison) [reply src/models-spec.js]: Doh, went a bit blind after starring at too much RDF/XML clearly. Have matched the error messages up in a later PR.

Comment (nickevansuk) [reply src/models-spec.js]: Super!

File: src/models-spec.js (hunk 1 above)
Comment (nickevansuk) [reply src/models-spec.js]: No worries

**Pull request merged**